### ISSUE TITLE
[BUG] Audio not working with webOS SMP

### ISSUE THREAD

**Issue by LucHermkens** [mariotaku/moonlight-tv#129]
**Describe the bug**

Audio is not working whatever audio backend I select when I'm using the webOS SMP video decoder. It has better performance on my TV than webOS NDL, but without audio working it has no use.

**Basic information**

 - Moonlight version: 1.2.8
 - Model: OLED65C16LA
 - webOS version: 6.2.0

**Comment by LucHermkens** [comment]
New discovery, sound works fine on TV speakers but not over (e)ARC. Is there any fix for this?

**Comment by mariotaku** [comment]
Are you using surround sound system? This is a long-lasting issue (see #49). In general if you are running webOS 5 or 6, NDL is preferred as it has lowest latency.

**Comment by LucHermkens** [comment]
Yes I do, is there any way to circumvent the 60 FPS framerate limit for the NDL decoder?

**Comment by mariotaku** [comment]
If you open Settings - About in moonlight, will you see your current refresh rate? It's your hardware limitation, and higher frame rate will not work.

**Comment by LucHermkens** [comment]
It's 120 Hz, which I can easily achieve with the SMP decoder, but not with the NDL decoder. Colors seem to be worse on the latter too, with all other settings equal.

**Comment by mariotaku** [comment]
I see. SMP + NDL audio is supposed to work if NDL decoder works though... NDL will lose some feature to achieve minimum latency.

**Comment by LucHermkens** [comment]
I tried forcing every audio decoder but without any luck. Audio works fine over TV speakers but not over (e)ARC unfortunately, the receiver doesn't even tell me there is any audio signal.

**Comment by mariotaku** [comment]
I'll add surround sound option later, so you can change and see if it works.
